infra/gcp/tests/main.tf

```diff
@@ -0,0 +1,23 @@
+module "project" {
+  source  = "terraform-google-modules/project-factory/google"
+  version = "~> 12"
+
+  name            = "Knative Tests"
+  project_id      = "knative-tests"
+  folder_id       = "1055082993535"
+  org_id          = "22054930418"
+  billing_account = "014273-543CE9-58CE57"
+
+  # Sane project defaults
+  default_service_account     = "keep"
+  disable_services_on_destroy = false
+  create_project_sa           = false
+  random_project_id           = false
+  auto_create_network         = true
+
+
+  activate_apis = [
```

Comment:
Are these the only APIs needed to be enabled for knative-tests project?

Reply:
This is useful for new projects as you need to enable the APIs before you can create resources from a particular API. In our case, the project has been around for a long time with relevant APIs already enabled.